Sources/HTTP/Request/Request.swift

```diff
@@ -0,0 +1,66 @@
+import Core
+import URI
+
+public final class Request: Message {
+    public var method: Method
+    public var uri: URI
+    public var version: Version
```

Comment:
@tanner0101 there's some issues later on w/ hardcoded versions ignoring this one, but aside from that allowing users the option should they need it is probably ok 👍